kola/tests/update/update.go

```diff
@@ -59,6 +65,22 @@ func Serve() error {
 	return omahawrapper.Serve()
 }
 
+// assert that -update does not fail
+// when there is nothing to do
+// https://github.com/kinvolk/Flatcar/issues/356
+func idle(c cluster.TestCluster) {
+	m := c.Machines()[0]
+
+	out, err := c.SSH(m, "update_engine_client -update")
+	if err != nil {
+		c.Fatalf("unable to run update_engine_client -update: %v", err)
+	}
+
+	if string(out) == "Update cancelled -- remote version is not newer than the local one" {
```

Comment:
I'm new to this, but just reading the logic I would think that the test should be:
```suggestion
	if string(out) != "Update cancelled -- remote version is not newer than the local one" {
```

Reply:
@jepio that's totally right, it seems my :brain: is already in week-end. :D 

I also added an output redirection in the fixup commit.